lib/peripheral.dart

```diff
@@ -91,6 +91,18 @@ class Peripheral {
         transactionId ?? TransactionIdGenerator.getNextId(),
       );
 
+  /// Discovers all [Service]s, [Characteristic]s and [Descriptor]s of this peripheral.
+  /// Must be done prior to any other operation concerning those.
+  ///
+  /// Optional [transactionId] could be used to cancel operation.
```

Comment:
The documentation here is incorrect. Let's let the user know that this is only available on Android.